Permission denied in bazel CI job

In https://paste.rs/Fl4 the Bazel job failed with this output:

```console
##[group]Run bazel run demo-rs --verbose_failures --noshow_progress
[36;1mbazel run demo-rs --verbose_failures --noshow_progress[0m
shell: /bin/bash -e {0}
##[endgroup]
2020/05/11 19:54:55 Downloading https://releases.bazel.build/3.1.0/release/bazel-3.1.0-linux-x86_64...
2020/05/11 19:54:55 could not run Bazel: could not start Bazel: fork/exec /home/runner/work/cxx/cxx/tools/bazel: permission denied
##[error]Process completed with exit code 1.
```

> "could not run Bazel: could not start Bazel: fork/exec /home/runner/work/cxx/cxx/tools/bazel: permission denied"

This job used to work and nothing has changed on our end. Still need to investigate.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -79,6 +79,7 @@ jobs:
           wget -q -O install.sh https://github.com/bazelbuild/bazel/releases/download/2.1.1/bazel-2.1.1-installer-linux-x86_64.sh
           chmod +x install.sh
           ./install.sh --user
+          echo ::add-path::$HOME/bin
       - name: Vendor dependencies
         run: |
           cp third-party/Cargo.lock .

